### ISSUE TITLE
All travis tests are suddenly failing

### ISSUE THREAD

**Issue by wbhart** [wbhart/flint2#807]
The relevant files haven't been touched for 28 days, but Travis fails after about a minute with [1]:

```
export LOCAL=$(pwd)/local
$ ./.build_dependencies
$ ./configure --with-mpir=${LOCAL} --with-mpfr=${LOCAL} --prefix=${LOCAL}
Invalid GMP directory
The command "./configure --with-mpir=${LOCAL} --with-mpfr=${LOCAL} --prefix=${LOCAL}" failed and exited with 1 during .
```

If anyone has any insight into this, it would be mighty helpful. I don't see anything wrong with what we have there, especially given that it was working earlier yesterday.

I've tried re-running the tests. Same result. I could try again, but that is the definition of madness.

[1] https://travis-ci.org/github/wbhart/flint2/builds/713511921

**Comment by thofma** [comment]
The travis log says on line 405 of the third job:
```
Resolving www.mpfr.org (www.mpfr.org)... 152.81.144.153
Connecting to www.mpfr.org (www.mpfr.org)|152.81.144.153|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://www.mpfr.org/mpfr-4.0.0/mpfr-4.0.0.tar.bz2 [following]
--2020-07-31 10:52:26--  https://www.mpfr.org/mpfr-4.0.0/mpfr-4.0.0.tar.bz2
Connecting to www.mpfr.org (www.mpfr.org)|152.81.144.153|:443... connected.
HTTP request sent, awaiting response... 503 Service Unavailable
2020-07-31 10:52:26 ERROR 503: Service Unavailable.
tar: mpfr-4.0.0.tar.bz2: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
```

**Comment by thofma** [comment]
TLDR: The mpfr server is down.

**Comment by wbhart** [comment]
Thanks @thofma 

It's really strange that it says invalid GMP directory. I just checked our configure to make sure that isn't what it prints when MPFR is missing. Maybe there's just a connectivity issue in general somewhere.

**Comment by thofma** [comment]
I think there are more broken things. MPIR is never properly configured or built (even if the flint2 run is succesful on travis). There is something broken with `yasm`, see https://travis-ci.org/github/wbhart/flint2/jobs/713399620#L403. It is always using gmp of the system, see https://travis-ci.org/github/wbhart/flint2/jobs/713399620#L7222.

**Comment by wbhart** [comment]
I think it is a travis problem. We didn't change anything at all that could do this.

I suspect the ${LOCAL} is not working. Probably they got up earlier in the morning than we did and changed the way everything works.

I'll see if it shakes loose in a few days.

**Comment by isuruf** [comment]
Building yasm has been broken for a while now. https://github.com/wbhart/flint2/pull/809 should fix it. MPFR website is down though.

**Comment by wbhart** [comment]
If so then Travis hasn't been informing me of test failures (or it's been using cache for ages).

But thanks for the PR. I may as well just merge it, as we can't test it until the server running France is back up.

**Comment by thofma** [comment]
Test failures here or in the MPFR repository? Here he just falls back to using the systems GMP.
